documentation/Usage/0_installation.rst

```diff
@@ -0,0 +1,28 @@
+Installation
+=============
+
+We use catkin to build our tools. So make sure that you have a working catkin workspace.
+
+Using command line
+------------------
+
+Clone the audio repository into the source folder of your catkin workspace.
+``git clone https://github.com/Roboy/roboy_audio /path/to/catkin_workspace/src/roboy_audio``
```

Comment:
what are the prerequisites/dependencies and how to get them (`sudo apt install`)